astropy/io/fits/tests/test_header.py

```diff
@@ -280,7 +280,7 @@ def __getitem__(self, key):
 
         header.update(NAXIS1=100, NAXIS2=100)
         assert set(header.keys()) == {"FOO", "A", "B", "HELLO", "NAXIS1", "NAXIS2"}
-        assert set(header.values()) == {"BAR", 1, 2, 100, 100}
+        assert tuple(header.values()) == ("BAR", 1, 2, 1, 100, 100)
```

Comment:
This does change the expected result for a FITS test. Should wait for @saimn to chime in here.

Reply:
It does change the expected result, but presumably towards the intended direction. The `set()` was removing the repeated values.

Reply:
But good to have this confirmed.

Reply:
Yeah. Now that I scroll a few lines up, I think indeed HELLO is set to 1. So I think this is good to merge. Thanks!